src/bcrypt/__init__.py

```diff
@@ -39,18 +39,19 @@ def gensalt(rounds=12, prefix=b"2b"):
     if prefix not in (b"2a", b"2b"):
         raise ValueError("Supported prefixes are b'2a' or b'2b'")
 
+    # TODO: is this the right min/max?
+    if rounds < 4 or rounds > 31:
+        raise ValueError("Invalid rounds")
+
     salt = os.urandom(16)
     output = _bcrypt.ffi.new("unsigned char[]", 30)
+    _bcrypt.lib.encode_base64(output, salt, len(salt))
```

Comment:
This one doesn't add trailing = but I don't know if there are any other differences. We may be able to switch (I originally had it using stdlib but converted it during the port for no good reason)